test/e2e/specs/editor-tracking/editor-tracking__block-inserted-event.ts

```diff
@@ -11,15 +11,14 @@ import {
 	PageTemplateModalComponent,
 	TestAccount,
 	EditorTracksEventManager,
-	TracksEventProperties,
 	SiteType,
-	TEST_LOCALES,
 	FullSiteEditorPage,
 	TemplatePartBlock,
 	OpenInlineInserter,
 	HeaderBlock,
 } from '@automattic/calypso-e2e';
 import { Browser, Page } from 'playwright';
+import type { TracksEventProperties } from '@automattic/calypso-e2e';
```

Comment:
This was/is a preference of mine, since I feel that doing this makes it explicit that this being imported as a type. 

I am primarily going off this response on StackOverflow: https://stackoverflow.com/a/64243357.